hardware_interface/source/hyperion.c

```diff
@@ -68,66 +68,76 @@ void hyperion_config_1_value(config_1_panel_t panel, config_1_channel_type_t cha
 
     switch (channel) {
     /* Channel 1 to 3 handle temp sensor */
-    case CONFIG_1_CHANNEL_TEMP_1:
+    case CONFIG_1_CHANNEL_TEMP_1:{
         adc_init(slave_addr, 1 << 7);
         adc_get_raw(slave_addr, &data, &ch); // invalid value
         adc_get_raw(slave_addr, &data, &ch); // correct value
-        *param = adc_calculate_sensor_temp(data, ADC_VREF);
+        int8_t *result = (int8_t *)param;
+        *result = (int8_t)(adc_calculate_sensor_temp(data, ADC_VREF));
         break;
-
-    case CONFIG_1_CHANNEL_TEMP_2:
+    }
```

Comment:
do we need curly brackets for switch cases?

Reply:
If you are going to use them there should be a space between the case and the {

Reply:
Done.